core/src/main/java/org/axonframework/commandhandling/model/LockingRepository.java

```diff
@@ -120,6 +136,23 @@ protected LockingRepository(Class<T> aggregateType, LockFactory lockFactory,
         this.lockFactory = lockFactory;
     }
 
+    /**
+     * Initialize the repository with the given {@code LockFactory}, {@code ParameterResolverFactory} and {@code
```

Comment:
Nit: I'd decapitalize the text within the `{@code}` blocks. I thought that would be the default for javadoc, as you're typically referencing parameters i.o. there types. However, I might be wrong there, so please correct me if so.